share/picongpu/examples/SingleParticleTest/include/picongpu/param/species.param

```diff
@@ -17,51 +17,71 @@
- *
- *  example: using UsedParticleShape = particles::shapes::CIC;
+/** Particle Shape definitions
+ *  Particle shape naming is according to
+ *    Hockney R, Eastwood JW. Computer simulation using particles. IOP Pubslishing Ltd (1988). p. 144.
+ *  That is, the name of a particle shape is defined by the visual form of its cloud shape S,
+ *  which describes a particle's charge density distribution.
+ *  However, the charge assigned to a mesh point due to a distribution of overlapping finite-sized
+ *  particles is obtained by evaluating and summing the assignment function W for all particles.
+ *  The assignment function is obtained by convolving the cloud shape with the top-hat function.
+ *  PIConGPU only implements the assignment functions.
+ *  **NOTE, the higher order shape names are redefined with release 0.6.0 in order to provide a consistent naming:
+ *      -> PQS is the name of the 3rd order shape (instead of PCS)
+ *      -> PCS is the name of the 4th order shape (instead of P4S)
+ *      -> P4S does not exist anymore
+ *  **
+ *  - particles::shapes::CIC : 1st order, assignment function is a piecewise linear spline
+ *  - particles::shapes::TSC : 2nd order, assignment function is a piecewise quadratic spline
+ *  - particles::shapes::PQS : 3rd order, assignment function is a piecewise cubic spline
+ *  - particles::shapes::PCS : 4th order, assignment function is a piecewise quartic spline
  */
 using UsedParticleShape = particles::shapes::CIC;
 
-/* define which interpolation method is used to interpolate fields to particle*/
-using UsedField2Particle = FieldToParticleInterpolation<UsedParticleShape, AssignedTrilinearInterpolation>;
+/** define which interpolation method is used to interpolate fields to particles
+ */
+using UsedField2Particle = FieldToParticleInterpolation<
+    UsedParticleShape,
+    AssignedTrilinearInterpolation
+>;
 
 /*! select current solver method
- * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
+ * - currentSolver::Esirkepov< SHAPE, STRATEGY > : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
  * - currentSolver::VillaBune< SHAPE, STRATEGY > : particle shapes - CIC (1st order) only
- * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PCS, P4S (1st to 4th order)
+ * - currentSolver::EmZ< SHAPE, STRATEGY >       : particle shapes - CIC, TSC, PQS, PCS (1st to 4th order)
  *
  * For development purposes:
- * - currentSolver::currentSolver::EsirkepovNative< SHAPE, STRATEGY > : generic version of currentSolverEsirkepov
+ * - currentSolver::EsirkepovNative< SHAPE, STRATEGY > : generic version of currentSolverEsirkepov
  *   without optimization (~4x slower and needs more shared memory)
  *
  * STRATEGY (optional):
- * - currentSolver::strategy::StridedCachedSupercells
- * - currentSolver::strategy::CachedSupercells
- * - currentSolver::strategy::NonCachedSupercells
+ * - currentSolver::StridedCachedSupercells
```

Comment:
Please revert this change it must be `currentSolver::strategy::StridedCachedSupercells`

Reply:
Wait, I was already asking about it for the very first version of this PR.